README.md

```diff
@@ -15,251 +18,329 @@ The role can be used to configure:
+one profile can be active on the device each time.
+For NetworkManager, a connection can only be active at one device each time.
+
+* For `NetworkManager`, the `name` option corresponds to the
+  [`connection.id`](https://developer.gnome.org/NetworkManager/stable/nm-settings.html#nm-settings.property.connection.id)
+  property option.
+  Although NetworkManager supports multiple connections with the same `connection.id`,
+  the `network` role cannot handle a duplicate `name`. Specifying a `name` multiple
   times refers to the same connection profile.
 
-* For initscripts, the name determines the ifcfg file name `/etc/sysconfig/network-scripts/ifcfg-$NAME`.
-  Note that here too the name doesn't specify the `DEVICE` but a filename. As a consequence
-  `'/'` is not a valid character for the name.
+* For `initscripts`, the `name` option determines the ifcfg file name `/etc/sysconfig/network-scripts/ifcfg-$NAME`.
+  Note that the `name` does not specify the `DEVICE` but a filename. As a consequence,
+  `'/'` is not a valid character for the `name`.
 
-### `state` and `persistent_state`
+You can also use the same connection profile multiple times. Therefore, it is possible to create a profile and activate it separately.
 
-Each connection profile can have a runtime state, represented by the `state`
-setting and a persistent state, represtented by the `persistent_state` setting.
+### `state`
 
-The optional `state` setting supports the following values:
+The `state` option identifies what is the runtime state of  each connection profile. The `state` option (optional) can be set to the following values:
 
-- `up`
-- `down`
+* `up` -  the connection profile is activated
+* `down` - the connection profile is deactivated
 
-It defines whether the profile is activated (`up`) or deactivated (`down`). If
-it is unset, the profile's runtime state will not be changed.
+#### `state: up`
+- For `NetworkManager`, this corresponds to `nmcli connection id {{name}} up`.
 
-The `persistent_state` setting is either `present` (default) or `absent`. If
-the `persistent_state` setting is `present` and the connection profile contains
-a `type` setting, the profile will be created or updated. If the profile is
-incomplete (lacks the `type` setting) and `persistent_state` is `present`,
-the behavior is undefined. The value `absent` makes the role ensure
-that the profile is not present on the target host.
+- For `initscripts`, this corresponds to `ifup {{name}}`.
 
+When the `state` option is set to `up`, you can also specify the `wait` option (optional):
 
-#### Example
+* `wait: 0` - initiates only the activation, but not wait until the device is fully connected.
+The connection is completed, for example after a DHCP lease received.
+* `wait: <seconds>` is a timeout that enables you to decide how long you give the device to
+activate. The default is using a suitable timeout. Note that the `wait` option is
+only supported by NetworkManager.
```

Comment:
The original version had
> **TODO** `wait` different from zero is not yet implemented.

@tyll @thom311 Is it intentional that it has disappeared?

Reply:
Yes: https://github.com/linux-system-roles/network/pull/86#discussion_r220863356